public/js/clients/chrome.js

```diff
@@ -17,13 +17,64 @@ Array.prototype.peekLast = function() {
 
 const bootstrap = require("./chrome/bootstrap");
 
-let connectionAgents;
+let currentConnection;
+let currentDebuggerAgent;
+let currentRuntimeAgent;
```

Comment:
I'm not completely sure about separating out the agents, but this made it so I could use it directly in `APIClient`. I suppose I could always call `getAgent("Debugger")` but this seemed like less work. I'm fine reverting this change though.